Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:8.9.0
```

Comment:
If this absolutely needs to go into the root folder can we add a comment that this is related to buildkite?